### DISCUSSION TITLE
Coverage plots for merged data

### DISCUSSION THREAD

**Discussion by danielcgingerich** [stuart-lab/signac#555] [Q&A] [answered]
This issue has just occurred to me.  In the vignettes, I see that coverage plots are only using data from one batch at a time.  I am making coverage plots from a merged data set of 24 different batches.  In my coverage plots, I am grouping the cells by disease versus control.  This equates to 12 separate disease samples, and 12 separate control samples.  Is the "normalized accessibility" of Tn5 insertions calculated separately for each dataset?  I understand that differences in sequencing depth could really mess up the coverage plots of merged datasets.

**Comment by timoast** [comment, marked as answer]
The normalization done on the coverage track normalizes for the number of cells in the group and the average total counts for that group, so differences in sequencing depth between groups are accounted for to some extent. You can see an example [here](https://satijalab.org/signac/articles/merging.html) where we create a coverage plot for a merged dataset of 4 different experiments with different average fragments per cell, and the resulting normalized tracks are quite similar. It's certainly not perfect, and if there are very large differences in sequencing depth between groups you might see some differences. These tracks are only used for visualization, and if you want to test for a significant difference in accessibility between peaks you should run a test using the `FindMarkers()` function.

**Reply by coopershawna** [reply]
To follow up on that, in the RNA space we had to be careful about testing for differential expression using the RNA assay vs the Integrated assay due to introducing dependencies.  How about in the ATAC space? Do we need to use merged vs integrated methods when we are interested in looking at differential accessibility of genomic regions between samples? We have a study with 1 cell type (hepatocytes) but separated by case vs control. Our samples are definitely separating by sample when merging and appear to integrate nicely when running the integration vignette. 
Thank you!  I really appreciate the ability to ask for advice!

**Reply by timoast** [reply]
You should use the original (merged) ATAC assay for differential accessibility, not the integrated assay. The reason is the same as for scRNA-seq data, the integration method introduces dependencies between datapoints that violate assumptions of the statistical tests. To compare between conditions you can cluster using the integrated data and then compare accessibility between conditions using the merged data.

**Reply by coopershawna** [reply]
Thank you!  Really appreciate the help!
